README.md

```diff
@@ -25,18 +29,19 @@ This DSL relies on the Ruby [pact-mock_service][pact-mock-service] gem to provid
   1. Install Karma using their [installation instructions](http://karma-runner.github.io/0.12/intro/installation.html)
 
     This basically consists of running,
-
-    * `npm install karma --save-dev`
-    * `npm install karma-jasmine karma-chrome-launcher --save-dev`
-    * `npm install -g karma-cli`
+    
+    ```
+    npm install karma karma-jasmine karma-chrome-launcher --save-dev
+    npm install -g karma-cli
+    ```
 
   1. Initialise and configure Karma
 
     Run `karma init`. Answer **jasmine** for *testing framework* and **no** for *use require.js*.
 
-  1. Add `pact-consumer-js-dsl` to your project by running `npm install DiUS/pact-consumer-js-dsl#X.Y.Z --save-dev`, where `X.Y.Z` is the latest stable version, according to the [releases][releases] page.
+  1. Add `pact-consumer-js-dsl` to your project by running `bower install pact-consumer-js-dsl --save-dev`.
 
```

Comment:
@bethesque If you're using Karma, it would need to use the web version, and not the node version .  For web libraries, you use Bower, for Node libraries you use npm.  It's not a hard rule, but either way, with the new UMD wrapping, they both work.